pontoon/sync/changeset.py

```diff
@@ -181,6 +181,9 @@ def get_entity_updates(self, vcs_entity, db_entity=None):
             'string_plural': vcs_entity.string_plural,
             'key': vcs_entity.key,
             'comment': '\n'.join(vcs_entity.comments),
+            'group_comment': '\n'.join(
+                vcs_entity.group_comments) if vcs_entity.group_comments is not None else '',
```

Comment:
Same as below: now that `vcs_entity.group_comments` is always a `list`, you can just use `'\n'.join(vcs_entity.group_comments)`.